src/content/en/fundamentals/media/fast-playback-with-video-preload.md

```diff
@@ -116,7 +116,9 @@ has actually been fetched.
 
 Setting the `preload` attribute to `metadata` indicates that the user is not
 expected to need the video, but that fetching its metadata (dimensions, track
-list, duration, and so on) is desirable.
+list, duration, and so on) is desirable. Note that starting in Chrome 62, the
+default value for `preload` is `metadata`. (It was `auto` previously).
```

Comment:
The feature is still disabled by default in Chrome 62. We are going to run an experiment to see the impact on users first.